composer/trainer/trainer_hparams.py

```diff
@@ -568,10 +571,16 @@ def initialize_object(self) -> Trainer:
             seed=seed,
             deterministic_mode=self.deterministic_mode,
 
-            # Callbacks and logging
+            # Callbacks
+            callbacks=callbacks,
+
+            # Logging
             run_name=self.run_name,
             loggers=loggers,
-            callbacks=callbacks,
+            progress_bar=self.progress_bar,
+            log_to_console=self.log_to_console,
+            log_level=self.log_level,
```

Comment:
For brevity, I called this `log_level`, though I realize that may lead to confusion. Should I call it `console_log_level`?

Reply:
(Going to call this console_log_level; I don't like log_level after looking at it haha)